schema_salad/python_codegen.py

```diff
@@ -215,9 +215,37 @@ def begin_class(
 """.format(
                     self.safe_name(name)
                 )
+        field_eqs = []
+        field_hashes = []
+        for name in field_names:
+            if name == "class":
+                field_eqs.append("self.class_ == other.class_")
+                field_hashes.append("self.class_")
+            else:
+                field_eqs.append("self.{0} == other.{0}".format(self.safe_name(name)))
+                field_hashes.append("self.{0}".format(self.safe_name(name)))
```

Comment:
should you always use `safe_name`? Otherwise you'd miss https://github.com/common-workflow-language/cwl-utils/blob/e315943d864dd127f18997212e4e3a93134686b9/cwl_utils/parser/cwl_v1_2.py#L12747 (`in_` from `WorkflowStep`)

Reply:
Yes, I just copied the constructor logic, but looking it again I see that it differentiates the `class_` field only because it must be populated with the `classname` string.

It is ok for me to unify all the fields using `safe_name`.